src/FSharp.Data.GraphQL.Client.DesignTime/ProvidedTypesHelper.fs

```diff
@@ -191,9 +152,8 @@ module internal ProvidedRecord =
             tdef.AddMembersDelayed(fun _ ->
                 let properties = propertiesGetter ()
                 let mapper (name : string, alias : string option, t : Type) = Option.defaultValue name alias, t
```

Comment:
Perhaps we could just use this as a lambda in `List.map`

Reply:
The code did change a bit because of overloaded params, but I made the change were it was needed.